__tests__/integration.test.js

```diff
@@ -137,3 +137,58 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("POST:201 - adds a comment to the given article", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({ username: "lurker", body: "riveting article" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: 0,
+          created_at: expect.any(String),
+          author: "lurker",
+          body: "riveting article",
+          article_id: 2,
+        });
+      });
```

Comment:
I'd recommend writing a 201 test to check that it ignores unnecessary properties. Like this:
```
{
username: "lurker", 
body: "riveting article",
extraProprties: "extra properties"
}
```